sql/synth_recipes.sql

```diff
@@ -3590,9 +3590,9 @@ INSERT INTO `synth_recipes` VALUES (3475,1,0,0,0,0,0,0,0,0,90,4099,4241,9487,0,0
 INSERT INTO `synth_recipes` VALUES (3476,1,0,0,0,0,72,0,0,0,0,4096,4238,610,633,4298,4570,0,0,0,0,5759,5759,5765,5765,6,8,6,8); -- red curry bun (cook 72) https://www.bg-wiki.com/bg/Red_Curry_Bun
 INSERT INTO `synth_recipes` VALUES (3477,1,0,0,0,0,98,0,0,0,0,4097,4239,4154,4441,5562,0,0,0,0,0,5411,5411,5411,5411,1,1,1,1); -- dawn mulsum
 INSERT INTO `synth_recipes` VALUES (3478,0,0,0,0,0,0,0,0,17,0,4100,4242,12560,0,0,0,0,0,0,0,649,850,852,852,4,4,1,1);
-INSERT INTO `synth_recipes` VALUES (3479,0,0,0,0,77,0,0,0,0,0,4098,4240,2150,2150,0,0,0,0,0,0,2649,2649,2649,2649,6,8,10,12);
-INSERT INTO `synth_recipes` VALUES (3480,0,2011,0,0,77,0,0,0,0,0,4098,4240,2145,2150,2150,2150,2150,2150,2150,0,2649,2649,2649,2649,18,24,30,36);
-INSERT INTO `synth_recipes` VALUES (3481,0,0,0,0,0,0,0,0,70,0,4096,4238,654,2538,2764,0,0,0,0,0,17762,17762,17762,17762,1,1,1,1); -- erlking's sword (smith 70) https://www.bg-wiki.com/bg/Erlking%27s_Sword
+INSERT INTO `synth_recipes` VALUES (3479,1,0,0,0,77,0,0,0,0,0,4098,4240,2150,2150,0,0,0,0,0,0,2649,2649,2649,2649,6,8,10,12);
+INSERT INTO `synth_recipes` VALUES (3480,1,2011,0,0,77,0,0,0,0,0,4098,4240,2145,2150,2150,2150,2150,2150,2150,0,2649,2649,2649,2649,18,24,30,36);
+INSERT INTO `synth_recipes` VALUES (3481,1,0,0,0,0,0,0,0,70,0,4096,4238,654,2538,2764,0,0,0,0,0,17762,17762,17762,17762,1,1,1,1); -- erlking's sword (smith 70) https://www.bg-wiki.com/bg/Erlking%27s_Sword
```

Comment:
1 = desynth in the current way its being (not actually) handled and 0 = synth. that sword has no desynth that I am seeing.

Reply:
I had that bass ackwards